JoelFaldin/ServerSort-MuniCrud-Backend
# Feat/add missing user functionalities

# Add missing user functionalities

## Description 🗒️✏️

Added missing route handlers for user management, correcting prisma schema. Now, the app is able to create a new user, change their role and delete ir from the user interface.

## How to test it? 🎯⚗️🔬

- Install the dependencies using `pnpm i`.
- For prisma, run `pnpx prisma migrate dev --name init`, then run `pnpx prisma generate`. Aditionally, you can start a local page to inspect the tables using `pnpx prisma studio`, but is not completely necessary.
- Run the frontend app, log in with admin privilegies, and check if creating a new user, changing their role and deleting it is possible via the frontend app.
